receiver/sqlqueryreceiver/receiver_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sqlqueryreceiver
+
+import (
+	"context"
+	"database/sql"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/consumer/consumertest"
+	"go.opentelemetry.io/otel/trace"
+	"go.uber.org/zap"
+)
+
+func TestCreateReceiver(t *testing.T) {
```

Comment:
Would it be useful to test failure scenarios as well? It looks like the only time we'd get an error would be from `scraperhelper.NewScraperControllerReceiver` so I'm not sure how important it is to test that.

Reply:
Yeah, I think I'm missing test coverage for errors. Will add.

Reply:
I've increased test coverage for error conditions (though not in receiver.go since scraper controller handles errors and that's tested separately).